wiki/Hit_Objects/en.md

```diff
@@ -60,6 +60,6 @@ Typical slider tick rate includes 1, 2 and 4. 3 is also available but used rarel
 
 ![A spinner in the default skin.](Spinner.jpg "A spinner in the default skin.")
 
-A spinner is a circle that takes up the entire height of the play area. Below the spinner is a number showing the spins per minute (RPM). The player should hold the mouse button down and spin in either a clockwise or anti-clockwise direction. As long as the mouse button is held down, the cursor is locked on the Spinner’s area, so the player need not worry about going outside it. Once the player has spinned for a while (dependent on the difficulty and length of the spinner), the spinner will be cleared, glowing white. Any continuous spins after this will award 1000 point bonuses per approximate spin. Spinning a spinner provides a continuous minor boost to the [Life Bar](/wiki/Glossary) depending on the spinning speed.
+A spinner is a circle that takes up the entire play area. Below the spinner is a number showing the spins per minute (RPM). The player should hold the mouse or keyboard button down and spin in either a clockwise or anti-clockwise direction. As long as the mouse button is held down, the cursor is locked on the Spinner’s area, so the player need not worry about going outside it. Once the player has spun for a while (dependent on the difficulty and length of the spinner), the spinner will be cleared, glowing white. Any continuous spins after this will award 1000 points bonuses per approximate spin. Spinning a spinner provides a continuous minor boost to the [Life Bar](/wiki/Glossary) depending on the spinning speed.
```

Comment:
```suggestion
A spinner is a circle that takes up the entire play area. Below the spinner is a number showing the spins per minute (RPM). The player should hold the mouse or keyboard button down and spin in either a clockwise or anti-clockwise direction. As long as the mouse button is held down, the cursor is locked on the Spinner’s area, so the player need not worry about going outside it. Once the player has spun for a while (dependent on the difficulty and length of the spinner), the spinner will be cleared, glowing white. Any continuous spins after this will award 1000-point bonuses per approximate spin. Spinning a spinner provides a continuous minor boost to the [Life Bar](/wiki/Glossary) depending on the spinning speed.
```